platform/api/aws/ec2.go

```diff
@@ -27,6 +28,21 @@ import (
 	"github.com/coreos/mantle/util"
 )
 
+// ListRegions lists the enabled regions in the AWS partition specified
+// implicitly by the CredentialsFile, Profile, and Region options.
+func (a *API) ListRegions() ([]string, error) {
+	output, err := a.ec2.DescribeRegions(nil)
+	if err != nil {
+		return nil, fmt.Errorf("describing regions: %v", err)
+	}
+	ret := make([]string, 0, len(output.Regions))
+	for _, region := range output.Regions {
+		ret = append(ret, *region.RegionName)
+	}
+	sort.Strings(ret)
```

Comment:
Hmm, any particular reason for sorting here rather than letting the caller do that if they need the list sorted?

Reply:
It seemed odd to return a list in random order.  But I can drop it if desired.

Reply:
Is there any particular value derived from the regions being in a sorted order? Nothing immediately struck out to me so I figured I'd ask if it's worth not spending the extra cycles sorting them always; it should never really become too cumbersome (unless AWS gets far too many regions) so I'm fine with keeping it if you feel there's value.